src/org/exist/util/serializer/XMLWriter.java

```diff
@@ -373,47 +373,72 @@ public void comment(final CharSequence data) throws TransformerException {
         }
     }
 
-    public void cdataSection(final char[] ch, final int start, final int len) throws TransformerException {
+    public void startCdataSection() throws TransformerException {
         if(tagIsOpen) {
             closeStartTag(false);
         }
-        
+
         try {
             writer.write("<![CDATA[");
-            writer.write(ch, start, len);
+        } catch(final IOException ioe) {
+            throw new TransformerException(ioe.getMessage(), ioe);
+        }
+    }
+
+    public void endCdataSection() throws TransformerException {
```

Comment:
should/could be private or package protected when required for unit tests

Reply:
Nope. It doesn't override anything.